spec/system/consumer/split_checkout_spec.rb

```diff
@@ -758,7 +758,8 @@
                 click_button("Apply")
 
                 expect(page).to have_content(
-                  "Your voucher value is more than your order. By using this voucher you are forfeiting the remaining value."
+                  "Your voucher value is more than your order. " \
+                  "By using this voucher you are forfeiting the remaining value."
```

Comment:
In my opinion you don't need to test for the whole text. Just testing for the key words is enough to assert that this text is displayed and it may allow little tweaks in the text later without updating the spec. And the spec becomes easier to read when you only test for the key content.